src/handlers/next/saleor-webhooks/saleor-async-webhook.ts

```diff
@@ -0,0 +1,20 @@
+import { NextApiHandler } from "next";
+
+import { AsyncWebhookEventType } from "../../../types";
+import { NextWebhookApiHandler, SaleorWebhook, WebhookConfig } from "./saleor-webhook";
+
+export class SaleorAsyncWebhook<TPayload = unknown> extends SaleorWebhook<TPayload> {
+  event: AsyncWebhookEventType;
+
+  protected type = "async" as const;
```

Comment:
```suggestion
  protected webhookType = "async" as const;
```
More exact name, to not confuse with the reserved keyword.